imagestreams/mariadb-centos.json

```diff
@@ -14,13 +14,13 @@
         "annotations": {
           "openshift.io/display-name": "MariaDB (Latest)",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a MariaDB database on CentOS. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mariadb-container/tree/master/10.3/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of MariaDB available on OpenShift, including major version updates.",
+          "description": "Provides a MariaDB database on CentOS. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mariadb-container/tree/master/10.5/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of MariaDB available on OpenShift, including major version updates.",
           "iconClass": "icon-mariadb",
           "tags": "database,mariadb"
         },
         "from": {
           "kind": "ImageStreamTag",
-          "name": "10.3-el8"
+          "name": "10.5-el8"
```

Comment:
There is no 10.5 for centos8. Do you mean the recently added `10.5-el7`?

Reply:
Good point. We will have soon quay.io/centos7/mariadb-105-centos7.

Reply:
Fixed